app/src/main/java/school/cactus/hnmobile/NewStoriesAdapter.kt

```diff
@@ -0,0 +1,52 @@
+package school.cactus.hnmobile
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import com.firebase.ui.database.FirebaseRecyclerAdapter
+import com.firebase.ui.database.FirebaseRecyclerOptions
+import com.google.firebase.database.DataSnapshot
+import com.google.firebase.database.DatabaseError
+import com.google.firebase.database.ValueEventListener
+import com.google.firebase.database.ktx.database
+import com.google.firebase.database.ktx.getValue
+import com.google.firebase.ktx.Firebase
+import school.cactus.hnmobile.NewStoriesAdapter.StoryItemHolder
+
+class NewStoriesAdapter(options: FirebaseRecyclerOptions<Int>) :
+    FirebaseRecyclerAdapter<Int, StoryItemHolder>(options) {
+
+    private val query = Firebase.database.reference
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): StoryItemHolder {
+        // Create a new view, which defines the UI of the list item
+        val view = LayoutInflater.from(parent.context)
+            .inflate(R.layout.item_news, parent, false)
+        return StoryItemHolder(view)
+    }
+
+    override fun onBindViewHolder(storyItemHolder: StoryItemHolder, position: Int, model: Int) {
+
+        val queryStory = query.child("/v0/item/$model").child("title")
+
+        queryStory.addListenerForSingleValueEvent(object : ValueEventListener {
+
+            override fun onCancelled(error: DatabaseError) {
+            }
+
+            override fun onDataChange(snapshot: DataSnapshot) {
+                val children = snapshot.getValue<String>()
+
+                storyItemHolder.textView.text = "News: "+children.toString()
```

Comment:
We should not forget to format our code before commiting.

Actually there is a setting for this in Android Studio.

<img width="271" alt="Screen Shot 2020-12-27 at 16 43 44" src="https://user-images.githubusercontent.com/4990386/103172178-b07e6380-4862-11eb-9c85-10cf964c8f45.png">

Reply:
thanks, noted.